MediaGalleryUi/view/adminhtml/templates/image_edit_details_standalone.phtml

```diff
@@ -53,6 +58,16 @@ use Magento\Backend\Block\Template;
                 }
             }
         },
+        "#media-gallery-image-edit-keywords-details": {
+            "Magento_Ui/js/core/app": {
+                "components": {
+                    "mediaGalleryEditKeywordsDetails": {
```

Comment:
I think it would be better to declare and initialize this component as a child to the edit details component. Please see `AdobeStockImageAdminUi/view/adminhtml/web/js/components/grid/column/image-preview.js` - `viewConfig`  for example